core-web/libs/sdk/experiments/src/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from './lib/hooks/useExperiments';
+export * from './lib/hooks/useExperimentVariant';
 export * from './lib/components/DotExperimentsProvider';
+export * from './lib/components/withExperiments';
 export * from './lib/contexts/DotExperimentsContext';
```

Comment:
Are you sure that you want your library to expose all this?

Reply:
No. Deleted all the unnecessary. Only the HOC is exposed now.